config/initializers/inflections.rb

```diff
@@ -0,0 +1,3 @@
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.singular(/PowerVirtualServers$/, 'PowerVirtualServers')
```

Comment:
There should probably be a singular inflection (what you have) and also a plural inflection (for going the other direction, otherwise Rails will drop the "s" thinking it should singularize.

I recommend adding some specs a la https://github.com/ManageIQ/manageiq/blob/master/spec/initializers/as_inflections_spec.rb#L1-L8

Reply:
Also I think we need to beef up that core spec 😆